### DISCUSSION TITLE
Can I just export the html and use in sphinx or django

### DISCUSSION THREAD

**Discussion by simkimsia** [markdoc/markdoc#78] [Q&A] [answered]
My question is similar to https://github.com/markdoc/markdoc/discussions/67 except I'm more proficient with python and I don't use browserify 

Is there a better way to use markdoc for non next or typescript developers?

**Comment by zeel01** [comment, marked as answer]
> Can I just export the html

If by this you mean run Markdoc on the `.md` files to generate `.html` files, then use those files as one would normally use HTML files... yes, if you write a little extra code. Unfortunately this would be JS/TS code by nature.

But it's fairly easy to achieve by using code from the examples, you would just read a file, run the content through Markdoc, and write it out to a new file. You could then even use Python's `subprocess` module to invoke this process as needed. Basically, you would create yourself a tiny CLI wrapper for Markdoc.

But since Markdoc is a JS/TS library, and doesn't have an existing CLI, you will end up having to write some JavaScript code.
